lib/manageiq/appliance_console/database_admin.rb

```diff
@@ -222,6 +276,15 @@ def file_menu_args
         ]
       end
 
+      def security_protocol_menu_args
+        [
+          "OpenStack Security Protocol",
+          SECURITY_PROTOCOL_OPTIONS,
```

Comment:
If this is the only place this is used, I think it would be better to just list the strings here, no need to declare a constant at the class level.

Reply:
yup.

Reply:
I can do that.